addons/io_sketchfab_plugin/blender/blender_version.py

```diff
@@ -4,50 +4,58 @@ class Version:
     """Adjusts functions according to the differences between 2.79 and 2.8"""
 
     #Render engine
-    ENGINE = "CYCLES" if bpy.app.version == (2, 79, 0) else "BLENDER_EEVEE"
+    ENGINE = "CYCLES" if bpy.app.version < (2, 80, 0) else "BLENDER_EEVEE"
```

Comment:
All these changes should be moved to previous commit with 2.79/2.80 compatibility, as they are no link with "Fixing various issues on animation import".
(A quick git trick can achieve that)

Also, it seems that after having moved all these compatibility changes, only fixes on materials should remain, so the commit should be renamed "Fixes various issues on materials". 

It will have no impact on the plugin itself, but it's important for the git history 👍